tests/test_tracing.py

```diff
@@ -14,22 +16,7 @@
     TraceHeader,
     XraySubsegment,
 )
-from datadog_lambda.tracing import (
-    _deterministic_md5_hash,
-    create_inferred_span,
-    extract_dd_trace_context,
-    create_dd_dummy_metadata_subsegment,
-    create_function_execution_span,
-    get_dd_trace_context,
-    mark_trace_as_error_for_5xx_responses,
-    set_correlation_ids,
-    set_dd_trace_py_root,
-    _convert_xray_trace_id,
-    _convert_xray_entity_id,
-    _convert_xray_sampling,
-    InferredSpanInfo,
-    extract_context_from_eventbridge_event,
-)
+import datadog_lambda.tracing as dt
```

Comment:
did this so we can importlib.reload the module, to force the parsing of DD_SERVICE_MAPPING for each test. (kinda a hacky way of doing this but it fixes the tests and lets us stay with the logic of only parsing DD_SERVICE_MAPPING when the tracing.py module is loaded.

Reply:
I fixed all other references with find and replace with the dt alias